cves/kernel/CVE-2020-36558.yml

```
@@ -284,8 +281,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    While this issue was primarily relating to concurrency, the threads were
+    apart of the same process.
```

```suggestion
  note: |
    While this issue was primarily relating to concurrency, the threads were
    a part of the same process.
```
Consider improving clarity.

---

changed